kivy/core/camera/camera_opencv.py

```diff
@@ -118,8 +118,10 @@ def init_camera(self):
             # get fps
             self.fps = self._device.get(PROPERTY_FPS)
 
-        if self.fps <= 0:
-            self.fps = 1 / 30.
+        if self.fps == 0 or self.fps == 1:
+            self.fps = 1 / 30
+        elif self.fps > 1:
+            self.fps = 1 / self.fps
```

Comment:
Same here, it should be `self.fps = 1.0 / self.fps`.

Reply:
Thanks, I'll put together a follow on pull request asap

Reply:
same for me, fine one mac, but on windows I needed the `self.fps = 1.0 / self.fps.`

Reply:
There was a follow on pull request from this one which made that change

Reply:
here it is https://github.com/kivy/kivy/pull/6032/files